proto/wippersnapper/uart/v1/uart.proto

```diff
@@ -36,6 +36,23 @@ message UARTDeviceAttachResponse {
   bool is_success  = 2; /** True if the UARTInitRequest was successful, False otherwise. */
 }
 
+/*
+* UARTDeviceDetachRequest represents a message sent from IO to a device
+* to detach a device from the UART bus.
+*/
+message UARTDeviceDetachRequest {
+    string device_id = 1[(nanopb).max_size = 15]; /** The unique identifier of the device to detach from the UART bus. */
+}
+
+/*
+* UARTDeviceDetachResponse represents a message sent from a device to IO to
+* confirm that a device has been detached from the UART bus.
+*/
+message UARTDeviceDetachResponse { 
+    string device_id = 1[(nanopb).max_size = 15]; /** The unique identifier of the device to detach from the UART bus. */
+    bool is_success  = 2; /** True if the UARTDeviceDetachRequest was successful, False otherwise. */
+}
+
```

Comment:
No need for this, I think. I don't know what the broker would do with it, and none of the other PBs use a detach response as far as i can see.